### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by LolGleb**
???

### CODE REVIEW

**Comment by FoxGleb** [comment]
Not bad, not bad

**Comment by FoxGleb** [comment]
Yo🙏
Nice👍

**Comment by FoxGleb** [comment]
lkmlknj jk   jk  f 
 efw
 df 
sd 
f s
f
 ewsef
 sd
 f
 sd
f s
d f
 
sdf 
sdf
 sd
 fds fd

File: tests/SomeClassTest.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 use PHPUnit\Framework\TestCase;
 
+require_once 'vendor/autoload.php';
```
Comment (FoxGleb) [inline tests/SomeClassTest.php:5]: Not sure if that really necessary

File: vendor/bin/php-parse [hunk 2]
```diff
@@ -1,119 +0,0 @@
-#!/usr/bin/env php
```
Comment (FoxGleb) [inline vendor/bin/php-parse:1]: WHYyyyyyyyy?

File: tests/SomeClassTest.php (hunk 1 above)
Comment (LolGleb) [reply tests/SomeClassTest.php:5]: IT IS

File: vendor/bin/php-parse (hunk 2 above)
Comment (LolGleb) [reply vendor/bin/php-parse:1]: Chikibamboni???

File: vendor/bin/php-parse.bat [hunk 3]
```diff
@@ -1,5 +0,0 @@
-@ECHO OFF
```
Comment (LolGleb) [inline vendor/bin/php-parse.bat:1]: Yo

Comment (LolGleb) [inline vendor/bin/php-parse.bat:1]: d

File: tests/SomeClassTest.php (hunk 1 above)
Comment (LolGleb) [inline tests/SomeClassTest.php:5]: Just a comment

File: vendor/bin/php-parse.bat [hunk 4]
```diff
@@ -1,5 +0,0 @@
-@ECHO OFF
-setlocal DISABLEDELAYEDEXPANSION
```
Comment (LolGleb) [inline vendor/bin/php-parse.bat:2]: Hop hey la la ley

File: vendor/bin/php-parse.bat (hunk 3 above)
Comment (LolGleb) [inline vendor/bin/php-parse.bat:1]: hop hey la la ley

File: vendor/bin/phpunit [hunk 5]
```diff
@@ -1,122 +0,0 @@
-#!/usr/bin/env php
```
Comment (LolGleb) [inline vendor/bin/phpunit:1]: cho

File: vendor/bin/php-parse.bat (hunk 3 above)
Comment (LolGleb) [inline vendor/bin/php-parse.bat:1]: d

**Review by FoxGleb — CHANGES_REQUESTED** [review]
Change somethingg

**Comment by LolGleb** [comment]
But I don't want to!

**Comment by FoxGleb** [comment]
Okaaaay

**Review by FoxGleb — APPROVED** [review]
All is fine

**Pull request merged**